oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/forms/hooks.js

```diff
@@ -448,3 +450,29 @@ export const useValidateOnBlur = () => {
 
   return handleValidateAndBlur(validateField, setFieldTouched);
 };
+
+export const useSanitizeInput = () => {
+  const {
+    formConfig: { allowedHtmlAttrs, allowedHtmlTags },
+  } = useFormConfig();
+
+  const sanitizeInput = useCallback(
+    (htmlString) => {
+      const decodedString = decode(htmlString);
+      const cleanInput = sanitizeHtml(decodedString, {
+        allowedTags: allowedHtmlTags,
+        allowedAttributes: allowedHtmlAttrs,
+      });
+      return cleanInput;
+    },
+    [allowedHtmlTags, allowedHtmlAttrs]
+  );
+
+  return {
+    sanitizeInput,
+    allowedHtmlAttrs,
+    allowedHtmlTags,
```

Comment:
why not also export a list of valid editor tags here by calling the newly created getValidTagsForEditor utility? :)

Reply:
this is to avoid constructing the list of tags in each & every component using this hook & editor